lang/LangPrimSource/SC_ComPort.cpp

```diff
@@ -111,127 +251,91 @@ void SC_UdpInPort::handleReceivedUDP(const boost::system::error_code& error, std
     }
 
     if (error) {
-        printf("SC_UdpInPort: received error - %s\n", error.message().c_str());
+        printf("UDP: received error - %s\n", error.message().c_str());
         startReceiveUDP();
         return;
     }
 
-    OSC_Packet* packet = (OSC_Packet*)malloc(sizeof(OSC_Packet));
-
-    packet->mReplyAddr.mProtocol = kUDP;
-    packet->mReplyAddr.mAddress = remoteEndpoint.address();
-    packet->mReplyAddr.mPort = remoteEndpoint.port();
-    packet->mReplyAddr.mSocket = udpSocket.native_handle();
+    std::unique_ptr<char[]> data(new char[bytesTransferred]);
```

Comment:
```suggestion
    auto data = std::make_unique<char[]>(new char[bytesTransferred]);
```

Reply:
Sadly doesn't work. `std::make_unique` doesn't have a variant that takes an existing pointer (makes sense, make_unique is supposed to do the allocating(), and a raw `char[]` doesn't have a constructor that takes the size of the array to allocate (which would fit the make_unique pattern).
Anyway, this should all be replaced with storage other than char[], but that's a bigger change that should be separated from this.